Ignition.Root/Ignition.Root.csproj

```diff
@@ -163,6 +163,7 @@
     </None>
     <None Include="Properties\PublishProfiles\Ignition.Root.pubxml" />
     <Content Include="Properties\SlowCheetah\SlowCheetah.Transforms.targets" />
+    <None Include="Properties\PublishProfiles\Local.pubxml" />
```

Comment:
Instead of adding a publish profile called _Local_, roll this change back and add a publish profile called _Ignition.Root_ to the _Ignition.Root_ project. This will keep the project clean of tons of dead publish profiles. Publish profiles are in the Git ignore file, so you don't have to worry about your publish settings being overwritten.